src/main/java/com/example/hello/hello/controller/HelloController.java

```diff
@@ -1,16 +1,24 @@
 package com.example.hello.hello.controller;
 
+import java.util.Map;
+import java.util.HashMap;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.ResponseBody;
+
 
 
 @RestController
 @RequestMapping("api/v1/hello")
 public class HelloController {
     @GetMapping
-    public String helloString() {
-        return "Hello World";
+    @ResponseBody
```

Comment:
why did you remove the `@GetMapping` annotation?

Reply:
Always make sure to follow already existing pattern when you are working on a code base, never re-invent the wheel until you know what is happening in and out of the system.

Reply:
Alright sir The @GetMapping is still there